package.json

```diff
@@ -1,31 +1,40 @@
 {
   "name": "assert",
-  "description": "The node.js assert module, re-packaged for web browsers.",
+  "version": "2.0.0-prerelease",
+  "description": "The assert module from Node.js, for the browser.",
+  "main": "build/assert.js",
+  "license": "MIT",
+  "homepage": "https://github.com/browserify/commonjs-assert",
+  "repository": "browserify/commonjs-assert",
+  "scripts": {
+    "build": "babel assert.js test.js --out-dir build && babel internal --out-dir build/internal && babel test --out-dir build/test",
```

Comment:
Is this a task to prepare `npm publish`? If so, you should use the `prepare` task:

https://docs.npmjs.com/misc/scripts#prepublish-and-prepare

So I can use this library by `npm install browserify/commonjs-assert` even in IE11, whereas other modern browsers does not require ES5 artifacts.

Reply:
Added, thanks/